README.md

```diff
@@ -14,8 +10,279 @@ But we need have a clear way to discover how you've been invited to a group with
+4A(4):::unreplicated
+dA(d)
+group-inviteA(group-invite):::groupInvite
+helsinkiA(helsinki):::unreplicated
+
+rootB(root)
+v1B(v1)
+9B(9)
+group-inviteB(group-invite):::groupInvite
+
+subgraph Staltz
+  rootA --> v1A --> 4A --> helsinkiA
+            v1A --> dA --> group-inviteA
+end
+
+subgraph Arj
+  rootB --> v1B --> 9B --> group-inviteB
+end
+
+classDef default stroke:none;
+classDef groupInvite fill: #BF2669, stroke:none, color:white;
+classDef group fill: #702A8C, stroke: none, color:white;
+classDef unreplicated opacity: 0.4, stroke: none;
+```
+_Diagram showing feed state of Arj and Staltz from Arj's perspective. The greyed out feeds show feeds that exist for Staltz but which Arj has yet to want to replicate_
+
+
+Assuming he accepts this invitation Arj then does the following:
+1. calculates the path to the "helsinki" group for staltz, and starts replicating it
+    - in this example `@statlz/v1/4/helsinki` (where "helsinki" is derived from the `groupKey`)
+2. creates a helsinki feed for themself
+
+
+```mermaid
+graph TB
+
+rootA(root)
+v1A(v1)
+4A(4) & dA(d)
+group-inviteA(group-invite):::groupInvite
+helsinkiA(helsinki):::group
+
+rootB(root)
+v1B(v1)
+9B(9) & cB(c)
+group-inviteB(group-invite):::groupInvite
+helsinkiB(helsinki):::group
+
+subgraph Staltz
+  rootA --> v1A --> 4A --> helsinkiA
+            v1A --> dA --> group-inviteA
+end
+
+subgraph Arj
+  rootB --> v1B --> 9B --> group-inviteB
+            v1B --> cB --> helsinkiB
+end
+
+classDef default stroke:none;
+classDef groupInvite fill: #BF2669, stroke:none, color:white;
+classDef group fill: #702A8C, stroke: none, color:white;
+```
+
+_Diagram showing updated state for Arj after he joins the group. Note the shards each feed lands in are random (but deterministic if you know the `groupKey`)_
+
+Staltz can see that Arj has accepted the invitation because he is able to decrypt the feed-announce message for Arj's "helsinki" feed, and read that the purpose is the `groupKey`.
+(He also knows where to watch for such a feed to be announced because the shared is deterministiclly chosen.)
+
+### 3. Non-group creator invites someone
+
+Arj now wants to invite Mix to the "helsinki" group. He follows the same pattern as in (2), but now as the inviter.
+
+Mix knows Arj is a part of the group because he was invited by them.
+Mix also knows staltz is part of the group because all `group/add-member` messages have
+```
+recps: [groupId, groupCreatorId, ...inviteeIds]
+```
+_As long as we know the creator we can always re-follow the chain of group-additions._
+
+:fire: TODO - we need to write the group spec up with these changes clearly somewhere.
+
+Staltz see Arj has invited Mix because he's replicating Arj's "group-invite" feed, so Statlz starts replicating Mix.
+
+
+
+## Questions
+
+1. if people in a group see that "mix was added" they are all going to start asking to replicate the feed "pathToMixsGroupFeed". On a network level this will cast a shadow revealing group members....
```

Comment:
```suggestion
```

We discussed in Signal that the `pathToMixGroupFeed` is not transmitted over network, only the `feedIdToMixGroupFeed` and from the feed ID alone you cannot infer what its purpose is.